src/frontend/src/flows/verifiableCredentials/index.ts

```diff
@@ -4,6 +4,7 @@ import { withLoader } from "$src/components/loader";
 import { showMessage } from "$src/components/message";
 import { showSpinner } from "$src/components/spinner";
 import { fetchDelegation } from "$src/flows/authorize/fetchDelegation";
+import { validateDerivationOrigin } from "$src/flows/authorize/validateDerivationOrigin";
```

Comment:
For a later PR: Given it is now used in multiple flows, this could be moved to some flow independent infrastructure place? Also the documentation in this file does not convey that it is used in multiple flows.

Reply:
Yep definitely! I also want to change the interface:
* the args are unnamed making it easy to swap the origins by mistake
* the validation should not be allowed to be called with an undefined derivationOrigin; handling that case should be the caller's responsibility

it's somewhere on my TODO list, but since we already pull some code from the `authorize` flow elsewhere I thought this wouldn't hurt